src/xml2js.coffee

```diff
@@ -166,7 +190,7 @@ class exports.Parser extends events.EventEmitter
     if @remaining.length <= @options.chunkSize
       chunk = @remaining
       @remaining = ''
-      # 
+      #
```

Comment:
That was my IDE auto-stripping whitespace. I have updated the PR to delete the line.